src/Parsehub/Parsehub.php

```diff
@@ -401,4 +416,40 @@ public function getProjectRunDeleteApiUrl($run_token)
         $url = $api_url . '/runs/' . $run_token . '?api_key=' . $api_key;
         return $url;
     }
+
+    /**
+    * Get http status code of url
+    * @param string $url url to check
+    * @return integer http status
+    */
+    public function getHttpStatusCode($url)
+    {
+        $handler = curl_init($url);
```

Comment:
$r = \Httpful\Request::head($url); 
not return any header informations. 
please look here http://laravel.io/bin/9vY89